src/servicedata.js

```diff
@@ -0,0 +1,15 @@
+export const serviceData = [
+  {
+    id: 1,
+    icon: '/images/lawn-mower.png',
+    title: 'Mowing',
+    description: 'Our lawn care crew offers meticulous mowing services to keep your lawn looking pristine and well-manicured. We ensure a consistent and even cut, leaving your grass healthy and vibrant. If desired, we can also edge the grass to keep it flush with sidewalks, buildings, and pathways. We offer one-time or periodic mowing services.'
+
+  },
+  {
+    id: 2,
+    icon: '/images/grabage.png',
+    title: 'Yard Cleanup',
+    description: 'Transform your outdoor space with our yard cleanup service. We’re available for garbage and junk removal from abandoned lots, apartment grounds, businesses, and residences. We take pride in creating a clean and organized environment that enhances the beauty of your property.'
+  }
+]
```

Comment:
Praise(nb): Great job using this strategy! This will help us have a "separation of concerns." Later when we update the content, we won't need to worry about messing up our presentation code.